pennylane/_qubit_device.py

```diff
@@ -81,6 +81,10 @@ class QubitDevice(Device):
             executions. A value of ``0`` indicates that no caching will take place. Once filled,
             older elements of the cache are removed and replaced with the most recent device
             executions to keep the cache up to date.
+        custom_decomps (Dict[Union(str, qml.operation.Operation), Callable]): Custom
+            decompositions to be applied by the device at runtime. When specified, the
+            device will create a custom expand function by combining the regular expansion
+            criteria with those specified by the custom decompositions.
```

Comment:
Sorry to be a pain, but could we add an example to the docstring? Although the description is accurate, it is so much quicker to see it!

Reply:
I meant to ask about this actually, I wasn't sure where it made sense to add this to the docs! Sure, I'll put it on `qml.device` page. @josh146 what do you mean by "update that docstring anyway to match our latest conventions"?

Reply:
Oh, just following https://pennylane.readthedocs.io/en/latest/development/guide/documentation.html#functions-and-methods, e.g., having the example _after_ the arguments etc.